client/src/components/Dashboard/Dashboard.js

```diff
@@ -41,6 +43,17 @@ const Dashboard = () => {
           />
         ))}
       </CardRowGrid>
+      <RowTitle>Newest Trailers</RowTitle>
+      <CardRowGrid>
+        {newestTrailersCardData.map((card, i) => (
```

Comment:
If we're pulling data from a JSON object we can make create an ID property in each obj to pass in as the key instead of an index.  We'll need one anyways that way we can use it as relational data when we tie in the database.  In fact, all the JSON objs should have a key add to them.